packages/db-kms/src/db-kms.js

```diff
@@ -165,13 +169,43 @@ const initDbKms = (fastify, kmsOptions = {}) => {
       );
     };
 
+    /**
+     * encrypt text using a secret
+     * @param {Record<string, unknown>} plainText the text to encrypt
+     * @param {Id} keyId the key id to encrypt with
+     * @returns {string} the encrypted text
+     */
+    const encrypt = async (plainText, keyId) => {
```

Comment:
rename it to `encryptText`. We may need to add `encryptBuffer` too. Either that or accept strings and buffers in the single encrypt method

Reply:
@sloops77 not totally sure which `node` API you are referring to, but as this function is just using the `crypto` package's `encrypt/decrypt`, I made a PR for buffer support in the `crypto` package: #320